src/prelude.rs

```diff
@@ -12,9 +12,12 @@
 //! data structures of the plonk library.
 
 pub use crate::{
-    circuit::{self, Circuit, PublicInputValue, VerifierData},
+    circuit::{
+        self, verify_proof, Circuit, FeIntoPubInput, GeIntoPubInput,
+        PublicInputValue, VerifierData,
+    },
     constraint_system::{ecc::Point, StandardComposer, Variable},
     error::Error,
-    proof_system::{Proof, VerifierKey},
-    proof_system::{Prover, ProverKey, Verifier},
+    proof_system::{Proof, ProverKey, VerifierKey},
```

Comment:
Don't we need ```Verifier``` here?

Reply:
I thought about exporting the `Verifier` and `Prover` abstractions.

But you can still access them via `plonk::proving_system::{Prover, Verifier}` and `prelude` should instead contain the things that should ideally be used by the user that wants to use the lib on the most common manner.

Reply:
Ok I see, then it should be fine